Source/MMHTMLParser.m

```diff
@@ -248,6 +248,7 @@ - (NSRange)_parseNameWithScanner:(MMScanner *)scanner
 {
     NSMutableCharacterSet *nameSet = [NSMutableCharacterSet alphanumericCharacterSet];
     [nameSet addCharactersInString:@":"];
+    [nameSet addCharactersInString:@"-"];
```

Comment:
Could you add this in the previous call to `addCharactersInString:` instead of adding a second call?